app/styles/app/modules/build-messages.scss

```diff
@@ -5,6 +5,51 @@
 
   font-size: $font-size-sm;
 
+  .header {
+    padding: 1em;
+
+    .icon-alert path,
+    .icon-error path {
+      stroke: $brick-red;
+    }
+    .icon-warn path {
+      stroke: $canary-yellow;
+    }
+
+    .tools {
+      float: right;
```

Comment:
Would it be feasible to change the layout to `flex`? `float`s usually need `clear`s, otherwise they can break layouts in different browsers.

Reply:
i couldn't get it behave using `flex` (you know, css skills from like 2005 still ;)

i've changed it to be positioned `absolute` and set the `top` and `right` margin to `1em` ... which i think works fine, too. please let me know if i should try harder with `flex` :)

Reply:
That should work too, thanks! 😄